internal/pkg/api/handleAck.go

```diff
@@ -740,3 +694,49 @@ func makeUpdatePolicyBody(policyID string, newRev, coordIdx int64) []byte {
 
 	return buf.Bytes()
 }
+
+func invalidateAPIKeys(ctx context.Context, zlog zerolog.Logger, bulk bulk.Bulk, toRetireAPIKeyIDs []model.ToRetireAPIKeyIdsItems, skip string) {
```

Comment:
moved this function outside of `AckT` so it is accessible from `handleCheckin` too